docling/backend/msexcel_backend.py

```diff
@@ -362,22 +362,22 @@ def _find_images_in_sheet(
                     #print(f"y-values: {series.yVal}")
 
                     print(f"xval type: {type(series.xVal).__name__}")
-                    
+
                     xvals = []
                     for _ in series.xVal.numLit.pt:
                         print(f"xval type: {type(_).__name__}")
                         if hasattr(_, 'v'):
                             xvals.append(_.v)
 
                     print(f"x-values: {xvals}")
-                            
+
                     yvals = []
                     for _ in series.yVal:
                         if hasattr(_, 'v'):
                             yvals.append(_.v)
-                            
-                    print(f"y-values: {yvals}")                    
-                    
+
```

Comment:
Let's remove the prints

Reply:
This is pre-existing code (just the whitespace was automatically sanitized by the IDE).
If you mean to refactor any pre-existing code, that can best be planned as a separate PR.